src/strings/en-US/PowerFxResources.resw

```diff
@@ -6262,4 +6262,8 @@
     <value>As is not permitted in this context</value>
     <comment>{Locked=As} This is an error message that shows up when the As keyword is used but is not valid</comment>
   </data>
+  <data name="ErrNamedFormula_MissingSemicolon" xml:space="preserve">
+    <value>Named formula must end with a semicolon.</value>
```

Comment:
Should "Named Formula" be translatable? @gregli-msft what do we normally do with names of features in these error messages?
Also, should it be capitalized?